tee-worker/app-libs/parentchain-interface/src/event_subscriber.rs

```diff
@@ -0,0 +1,60 @@
+/*
+	Copyright 2021 Integritee AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use itp_api_client_types::ParentchainApi;
+use itp_types::parentchain::{BalanceTransfer, ParentchainId};
+use substrate_api_client::SubscribeEvents;
+
+pub fn subscribe_to_parentchain_events(api: &ParentchainApi, parentchain_id: ParentchainId) {
```

Comment:
what's this one for? if I follow correctly, it subscribes forever and log event's info only

Reply:
It's been like this since the beginning 😂  for now it's used in tests and CLI.
My understanding is it prefaces the upcoming change where parentchain events should be used to trigger indirect calls instead of extrinsics => even if upstream doesn't do it we might want it